content/en/docs/started/architecture.md

```diff
@@ -4,142 +4,147 @@ description = "An overview of Kubeflow's architecture"
+  preparation and feature engineering step.
+
+- [Kubeflow Notebooks](/docs/components/notebooks/) can be used for model development and interactive
+  data science to experiment with your ML workflows.
 
-For details of the workflow and to run the system yourself, see the 
-[end-to-end tutorial for Kubeflow on GCP](https://github.com/kubeflow/examples/tree/master/mnist#mnist-on-kubeflow-on-gcp).
+- [Kubeflow Katib](/docs/components/katib/) can be used for model optimization and hyperparameter
+  tuning using various AutoML algorithms.
 
-<a id="interfaces"></a>
-## Kubeflow interfaces
+- [Kubeflow Training Operator](/docs/components/training/) can be used for large-scale distributed
+  training or fine-tuning.
+
+- [Kubeflow Model Registry](/docs/components/model-registry/) can be used to store ML metadata,
+  model artifacts, and preparing models for production serving.
+
+- [KServe](https://kserve.github.io/website/master/) can be used for online and batch inference
+  in the model serving step.
+
+- [Feast](https://feast.dev/) can be used as a feature store and to manage offline and online
+  features.
+
+- [Kubeflow Pipelines](/docs/components/pipelines/) can be used to build, deploy, and manage each
+  step in the ML lifecycle.
+
+You can use most Kubeflow components as
+[standalone tools](/docs/started/introduction/#what-are-standalone-kubeflow-components) and
+integrate them into your existing AI/ML Platform, or you can deploy the full
+[Kubeflow Platform](/docs/started/introduction/#what-is-kubeflow-platform) to get all Kubeflow
+components for an end-to-end ML lifecycle.
+
+## Kubeflow Interfaces
 
 This section introduces the interfaces that you can use to interact with
 Kubeflow and to build and run your ML workflows on Kubeflow.
 
-### Kubeflow user interface (UI) 
+### Kubeflow User Interface (UI)
 
 The Kubeflow UI looks like this:
 
 <img src="/docs/images/central-ui.png" 
   alt="The Kubeflow UI"
   class="mt-3 mb-3 border border-info rounded">
 
-The UI offers a central dashboard that you can use to access the components
-of your Kubeflow deployment. Read 
-[how to access the central dashboard](/docs/components/central-dash/overview/).
+The Kubeflow Platform includes [Kubeflow Central Dashboard](/docs/components/central-dash/overview/)
+which acts as a hub for your ML platform and tools by exposing the UIs of components running in the
+cluster.
 
-## Kubeflow APIs and SDKs
+### Kubeflow APIs and SDKs
 
-Various components of Kubeflow offer APIs and Python SDKs. See the following
-sets of reference documentation:
+<!--
+TODO (andreyvelich): Add reference docs once this issue is implemented: https://github.com/kubeflow/katib/issues/2081
+-->
+
+Various components of Kubeflow offer APIs and Python SDKs.
 
-* [Pipelines reference docs](/docs/components/pipelines/reference/) for the Kubeflow
+See the following sets of reference documentation:
+
```

Comment:
I think we can remove the spaces between these bullet lins, as the double-spacing makes it harder to read.